### PR TITLE
Add gyazo.com

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Gyazo is a screenshot and image sharing platform that allows users to capture, upload, and share images or GIFs instantly. The platform provides 2FA for the account login verification process. The website is ranked 7,255 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=gyazo.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Here's an SVG icon you could use:
icon

File: entries/g/gyazo.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Gyazo": {
+    "domain": "gyazo.com",
+    "img": "gyazo.com.png",
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://help.gyazo.com/Two-step%20login%20verification-5ed8969fc1eca80023e84383",
```
Comment (Carlgo11) [inline entries/g/gyazo.com.json]:
```suggestion
    "documentation": "https://help.gyazo.com/--5ed8969fc1eca80023e84383",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/g/gyazo.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "Gyazo": {
+    "domain": "gyazo.com",
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://help.gyazo.com/--5ed8969fc1eca80023e84383",
+    "categories": [
+      "creativity"
```
Comment (Carlgo11) [inline entries/g/gyazo.com.json]: From what I understand, Gyazo is a file sharing service, akin to Dropbox. If that's the case, _Backup and Sync_ would be a better fit.
```suggestion
      "backup"
```

Comment (Yang-Nankai) [reply entries/g/gyazo.com.json]: @Carlgo11 I see. Please forgive me for not being familiar with website classification. I thought this website was a service for sharing creative inspiration. However, I took a closer look and you are right. We are currently researching websites about 2FA, so we have sorted out a lot of websites. I am not very familiar with Github submissions, so thank you for correcting me in time! I have committed it again, hope your review!

**Comment by Yang-Nankai** [comment]
I would also like to ask why there are sometimes problems with the _Action_ part in _External Tests_, but I tested the _curl_ statement and there is no problem locally. When I submit, do I need to ensure that both _Actions_ pass before requesting a review?

**Comment by Carlgo11** [comment]
@Yang-Nankai Error status 429 means the API is throttling requests. It usually happens when a lot of commits are made within a short time period.

It doesn't affect the ability for the PR to be merged.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**